docs/building_blocks_for_flexible_content.md

```diff
@@ -305,6 +305,11 @@ A hero block renders a vertical column of blocks in a two-thirds-width block on
 
 ##### Example
 
+The YAML structure for hero images is slightly different depending on whether you're looking at the content item in Content Store (or a hardcoded YAML block which has to match the Content Store format),
+or in Whitehall.
```

Comment:
nitpick: unnecessary newline?

Reply:
Fixed in the force push (the line was just a bit too long for my tastes, but happy to have it on one line)